app/pb_kits/playbook/pb_checkbox/_checkbox.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= content_tag(:label, id: object.id, data: object.data, value: object.value, name: object.name, class: object.classname(object.kit_class)) do %>
```

Comment:
I do not think the `value` and `name` is needed on the label element. Is there a reason for this that I may have not thought about?

Reply:
Value and name should be on the checkbox